File: PHP/DDDTest.php
<?php
/**
 * @version PHP
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ ./vendor/bin/phpunit NomeDaClasseTest.php
 */
namespace PHP;

use PHPUnit\Framework\TestCase;
require ("DDD.php");

class DDDTest extends TestCase {

    public function testDDD() {
        $obj = new DDD();
        //Test 1
        $want = "Sao Paulo";
        $got = $obj->dddCity(11);
        $this->assertEquals($want, $got);
        //Test 2
        $want = "DDD nao cadastrado";
        $got = $obj->dddCity(13);
        $this->assertEquals($want, $got);
    }

}

File: JAVA/DDDTest.java
/** 
 * @version JAVA
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ mvn clean test -Dtest=your.package.TestClassName
 */ 
package JAVA;

import org.junit.Assert;
import org.junit.Test;

public class DDDTest {

	@Test
	public void testDDD() throws Exception {
		//Test 1
		String want = "Sao Paulo";
		String got = DDD.dddCity(11);
    Assert.assertEquals(want, got);
		//Test 2
		want = "DDD nao cadastrado";
		got = DDD.dddCity(13);
		Assert.assertEquals(want, got);
	}

}
